thrust/examples/arbitrary_transformation.cu

```diff
@@ -62,29 +60,20 @@ struct arbitrary_functor2
 
 int main()
 {
-  // allocate storage
-  thrust::device_vector<float> A(5);
-  thrust::device_vector<float> B(5);
-  thrust::device_vector<float> C(5);
+  // allocate and initialize storage using initializer lists
```

Comment:
I believe the second part does not add anything, technically this is also termed `direct-list-initialization`, see https://en.cppreference.com/w/cpp/language/list_initialization.html
```suggestion
  // allocate and initialize storage
```